taichi/transforms/vector_split.cpp

```diff
@@ -241,8 +242,11 @@ class BasicBlockVectorSplit : public IRVisitor {
 
   void visit(PrintStmt *stmt) override {
     for (int i = 0; i < current_split_factor; i++) {
-      current_split[i] =
-          Stmt::make<PrintStmt>(lookup(stmt->stmt, i), stmt->str);
+      std::vector<Stmt *> new_contents;
+      std::transform(stmt->contents.begin(), stmt->contents.end(),
```

Comment:
I just realized that this is wrong: `std::transform` will not automatically call `push_back` for `new_contents`. You need to either make sure `new_contents`'s size is the same as `stmt->contents`, or you can try [`std::back_inserter`](https://en.cppreference.com/w/cpp/iterator/back_inserter). Example in https://en.cppreference.com/w/cpp/algorithm/transform